IRremoteESP8266.cpp

```diff
@@ -472,20 +472,16 @@ void IRsend::sendDenon (unsigned long data,  int nbits) {
   space(0);  // Always end with the LED off
 }
 
-void IRsend::mark(unsigned int time) {
+void IRsend::mark(unsigned int usec) {
```

Comment:
I will, when I clean up space(). Geesh. Wants small commits with only one thing, then asks for two things in a commit. ;-P